DTS-STN/secure-client-hub
# Fix dev pipeline

## [ADO-137803](https://dev.azure.com/VP-BD/DECD/_workitems/edit/137803)

Fixed [AB#137803](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/137803)

### Description of proposed changes:
This PR opts to use Next.js redirects in place of the middleware since something is inherently broken with how the middleware is handling requests. The middleware will be revisited at a later time since we will eventually need to add logging and tracing, but the redirects function does effectively the same thing at this time.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/en` or `/fr` and ensure you are redirected to `/`
4. Navigate to `/my-dashboard` and ensure that you are redirected to `/en/my-dashboard`
